### ISSUE TITLE
Author information and column

### ISSUE THREAD

**Issue by fmoralesc** [burtonator/polar-bookshelf#671]
It seems like it would be useful to keep information about the documents authors, if available. 

There should be a way to add author information to documents and show it in a dedicated column.

**Comment by burtonator** [comment]
Agreed.. We're going to add more info to the DocInfo object additionally for keeping bibliography information and resolving the DOI of the document to the full doc metadata.

We will give you the option to expose this information and make the column visible but not sure it will be visible by default.

Is this for academic use or  you're just generally interested in this column?

**Comment by fmoralesc** [comment]
For academic use. I'm using polar to organize papers now, and it was the
first thing I missed. Of course, a more general solution like you suggest
would be even better.

**Comment by burtonator** [comment]
Gotcha.  Are you coming from an existing tool?  Are you needing this feature to generate citations?

**Comment by fmoralesc** [comment]
I've tried other tools like this in the past (Mendeley is the one I used the most), but I didn't commit to any of them. I like the simplicity of polar in comparison, and I think I will stick to it.

I don't really need the feature for citations at the moment (I use bibtex, and I kind of prefer to build my bibliographies by hand). In my experience I tend to organize myself around titles and author names mainly, so what I'm doing now is to add the names to the titles so I can at least search for them. However, I know that if at some point I would like to use this metadata to extract citations, I will have to preprocess it. Not a big deal, in any case, just a convenience thing. 

As far as I am concerned, if you are planning to allow a more flexible metadata management scheme in the future, that's good enough for me.

**Comment by clemux** [comment]
I've been using polar for a few weeks now for literature review while writing a paper, and access to the author metadata (also year of publication) is a crucial feature for retrieving papers quickly. I'm currently having to resort to manually add tags with author names, which might become a mess really fast.

Please consider adding a manual way to edit the author information before implementing full DOI/metadata retrieval if it means having that information at least a week earlier.

**Comment by burtonator** [comment]
Gotcha. This use case is becoming more important so I'll probably look at adding csl json this weekend. I'm also working on a new sidebar and dock to display the information properly.
